java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/h2/JdbcToArrowTimeZoneTest.java

```diff
@@ -129,6 +121,25 @@ public void testJdbcToArrowValues() throws SQLException, IOException {
                     Calendar.getInstance(TimeZone.getTimeZone(table.getTimezone())))
                 .build()),
         false);
+    testDataSets(
+        sqlToArrow(
+            conn.createStatement().executeQuery(table.getQuery()),
+            new RootAllocator(Integer.MAX_VALUE),
+            Calendar.getInstance(TimeZone.getTimeZone(table.getTimezone()))),
+        false);
+    testDataSets(
+        sqlToArrow(
+            conn.createStatement().executeQuery(table.getQuery()),
+            Calendar.getInstance(TimeZone.getTimeZone(table.getTimezone()))),
+        false);
+    testDataSets(
+        sqlToArrow(
+            conn.createStatement().executeQuery(table.getQuery()),
+            new JdbcToArrowConfigBuilder(
+                    new RootAllocator(Integer.MAX_VALUE),
+                    Calendar.getInstance(TimeZone.getTimeZone(table.getTimezone())))
+                .build()),
+        false);
```

Comment:
could I get bit of an explanation about this part?

Reply:
Good catch. It was my mistake. I have removed it.